src/utils/updateDatabase.ts

```diff
@@ -46,3 +31,48 @@ export async function updateDatabase(collections: Collection[]) {
     /* tslint:enable:no-unused-expression */
   }
 }
+
+const upsert_data = async (collection, collections) => {
+  try {
+    for (const entry of collections) {
+      // if an entry's `price_guidance` is left null in the dataset, null
+      // it out in the database - after this is finished we will compute price
+      // guidance for all collections that have a null value for it.
```

Comment:
I think this makes sense. To recap, the price guidance column in the sheet has these semantics:

- a blank always indicates "gimme the computed value"
- a value always indicates an override to be used instead of the computed value

Therefore, on first import:
- blanks result in the computed value
- values result in the override

And on re-import:
- blanks result in a _newly_ computed value, whether the blank was always there or newly blanked out
- values result in the override

If correct, this seems 👍 to me. (Upon re-reading, I think the integration spec confirms exactly this,  lmk if that's not right)